topics/ui/controls/checkbox.topic

```diff
@@ -0,0 +1,207 @@
+                                <td width="50%">
+                                        <format color="E55765" style="bold">Incorrect</format><img src="checkbox_writing_2_incorrect.png"
+                                                                                                   alt="An incorrect checkbox with the declarative form of the verb" width="378"/>
+                                </td>
+                            </tr>
+                        </table>
+                    </li>
+                    <li>Use the imperative form of verbs:
+                        <table style="none" border="false">
+                            <tr>
+                                <td width="50%">
+                                        <format color="369650" style="bold">Correct</format><img src="checkbox_writing_3_correct.png"
+                                                                                                 alt="A correct checkbox with the imperative form of the verb" width="378"/>
+                                </td>
+                                <td width="50%">
+                                        <format color="E55765" style="bold">Incorrect</format><img src="checkbox_writing_3_incorrect.png"
+                                                                                                   alt="An incorrect checkbox with the declarative form of the verb" width="378"/>
+                                </td>
+                            </tr>
+                        </table>
+                    </li>
+                    <li>Do not use negation in labels as it complicates understanding:
+                        <table style="none" border="false">
+                            <tr>
+                                <td width="50%">
+                                        <format color="369650" style="bold">Correct</format><img src="checkbox_writing_4_correct.png"
+                                                                                                 alt="A correct checkbox with the imperative form of the verb" width="378"/>
+                                </td>
+                                <td width="50%">
+                                        <format color="E55765" style="bold">Incorrect</format><img src="checkbox_writing_4_incorrect.png"
+                                                                                                   alt="An incorrect checkbox with the declarative form of the verb" width="378"/>
+                                </td>
+                            </tr>
+                        </table>
+                        <note>Exception: <control>Do not show again</control> checkbox.</note>
+                    </li>
+                    <li>Make labels short and intelligible. See <a href="writing_short.md">Writing short and clear</a>.
+                    </li>
+                </list>
+        </chapter>
+    </chapter>
+    <chapter title="Placement" id="placement">
```

Comment:
Possibly rename to "How to layout" and give link to https://plugins.jetbrains.com/docs/intellij/layout.html#checkboxes-and-radio-buttons

Reply:
Thanks, I agree! Plus, it will be just one link and the chapter would cover everything